rules/windows/defense_evasion_execution_windefend_unusual_path.toml

```diff
@@ -1,7 +1,7 @@
 [metadata]
 creation_date = "2021/07/07"
 maturity = "production"
-updated_date = "2021/07/07"
+updated_date = "2021/09/22"
 
 [rule]
 author = ["Elastic"]
```

Comment:
I think this is a big enough addition to the rule

```
author = ["Elastic", "Dennis Perto"]
```

Reply:
Makes sense to me @LaZyDK

Reply:
@LaZyDK, can you update this one so we can merge? Thanks!